contrib/examples/example_add_bot/setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import find_packages, setup
+
+setup(
+    name='intelmqExampleAddBot',
+    version=3.1,  # noqa: F821
+    maintainer='Sebastian Wagner',
+    maintainer_email='intelmq-dev@lists.cert.at',
```

Comment:
I don't want to be responsible for code which I never wrote and never saw =)
```suggestion
    maintainer='Your Name',
    maintainer_email='you@example.com',
```